docs/orchestrating-elastic-stack-applications/elasticsearch/storage-recommendations.asciidoc

```diff
@@ -0,0 +1,66 @@
+:parent_page_id: elasticsearch-specification
+:page_id: storage-recommendations
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{parent_page_id}.html#k8s-{page_id}[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Storage recommendations
+
+ECK does not come with its own storage mechanism for Elasticsearch data. It is compatible with any Kubernetes storage option. It is recommended to use PersistentVolumes, by configuring the <<{p}-volume-claim-templates,VolumeClaimTemplates>> section of the Elasticsearch resource.
+
+Multiple PersistentVolume storage classes are available, depending on your Kubernetes setup. Their specifications impact Elasticsearch performance and operations. There is no best solution to suit everyone: it is important to understand the trade-offs between various available options.
+
+[float]
+== Network-attached or Local PersistentVolumes
+
+PersistentVolumes come in two flavours, with a different performance/price/operations trade-off. ECK handles both the same way.
+
+**Network-attached PersistentVolumes** can generally be attached to a Pod regardless of the host they are scheduled on.
+They provide a major operational benefit: if the host goes down, or needs to be replaced, the Pod can simply be deleted. Kubernetes reschedules it automatically on a different host (generally in the same region), and reattaches the same volume. This can take only a few seconds, and does not require any human intervention.
+
+**Local PersistentVolumes** are bound to a particular host, and generally map a directory on the filesystem. Once bound to a Local PersistentVolume, a Pod can only be scheduled on the same host. If that host goes down, or needs to be replaced, the Pod cannot be scheduled on a different host. It remains in a `Pending` state until the host is available, or until the PersistentVolumeClaim is manually deleted. For that reason, Local PersistentVolumes bring more operational overhead.
```

Comment:
I wonder if it's worth going into more detail on recovering from a node failure with local PVs

Reply:
that's what I try to do (briefly though) in the `Local PersistentVolumes operations` section

Reply:
++ I got to that later. Maybe link to that section here and expand it? It seems to trip people up so I think it's worth writing it out in detail once so we can point people to it